Configurations/unix-Makefile.tmpl

```diff
@@ -654,9 +654,11 @@ errors:
 	( cd $(SRCDIR); $(PERL) util/ck_errf.pl -strict */*.c */*/*.c )
 	( cd $(SRCDIR); $(PERL) util/mkerr.pl -internal )
 	( cd $(SRCDIR)/engines; \
-	  for e in *.ec; do \
-	      $(PERL) ../util/mkerr.pl -conf $$e -rebuild *.c; \
-	  done )
+          $(PERL) ../util/mkerr.pl -conf e_capi.ec e_capi.c ; \
+          $(PERL) ../util/mkerr.pl -conf e_dasync.ec e_dasync.c ; \
+          $(PERL) ../util/mkerr.pl -conf e_ossltest.ec e_ossltest.c ; \
+          cd afalg; \
+          $(PERL) ../../util/mkerr.pl -conf e_afalg.ec e_afalg.c )
```

Comment:
Uhmm, they worked perfectly for all files directly in `engines/`...  I just tested by removing stuff from the diverse `engines/*_err.h`, and they got regenerated without issue.  `engine/afalg/` is left unprocessed, though, but that's quite easy to resolve, here's how:

```
( cd $(SRCDIR)/engines;
  find . -name '*.ec' | while read E; do $(PERL) ../util/mkerr.pl -conf $$E `dirname $$E`/*.c; done )
```